3717_Sprint/JohnDB_1/queries.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE exams (
+	name VARCHAR(255),
+	score INTEGER,
+	exam_date DATE);
+	
```

Comment:
You should probably create an auto increment field for primary key.

Reply:
The "How to create tables" knowledge is placed somewhere at the end of the book. It is going to be a composite primary key here. I don't know how to do it yet